stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/components/AbstractStub.java

```diff
@@ -86,7 +83,11 @@ public void setDescriptors(Set<Descriptor> descriptors) {
     }
 
     public void setMessageList(List<Message> messageList) {
-        this.messageList = messageList;
+        Map<String, Message> messageMap = new HashMap<>();
+        for (Message messageInList : messageList) {
```

Comment:
Rename `messageInList ` -> `message` to improve the readability.

Reply:
I renamed the variable name